packages/webapp/src/members/api/fixtures.ts

```diff
@@ -7,9 +7,11 @@ export const fixtureEdenMembers: EdenMember[] = [
         account: "alice.edev",
         nft_template_id: 147800,
         status: MemberStatus.ActiveMember,
-        election_participation_status: ElectionParticipationStatus.NoDonation,
+        election_participation_status: ElectionParticipationStatus.NoDonation, // See ElectionParticipationStatus for enum values; NoDonation is kinda the default if other special cases don't apply
```

Comment:
According to [Sparky's PR #380 on this line](https://github.com/eoscommunity/Eden/pull/380/files#diff-b7c8a9e6b2d5029dc73cc11ca3055423d65c53b04cc9e6dae57ed20ade050c31R2), we now only have two statuses: `NotInElection` and `InElection`. And the default (which he updates on these fixtures in his PR) is `NotInElection`. This make sense with the removal of the election fee. FYI.

Reply:
Cool. looks like contract was updated. Looks like his merge will update the file.